tests/helper/tests/users.py

```diff
@@ -51,3 +55,24 @@ def users(client):
         if line != '':
             homes.append(line)
     assert len(homes) == 0, f"Found the following home directories: {homes}"
+
+
+def _has_user_sudo_cmd(client, user):
+    """ Check if user has any sudo permissions """
+    # Execute command on remote platform
+    cmd = f"sudo -l -U {user}"
+    out = utils.execute_remote_command(client, cmd)
+
+    # Write each line as output in list
+    output_lines = [] 
+    for line in out.split("\n"):
+       output_lines.append(line)
+
+    # Check if there is enough content in our list
+    # and validate if there are related sudo commands
+    sudo_cmd = False
+    if len(output_lines) > 3:
+        if "may run the following commands on" in output_lines[-2]:
+           sudo_cmd = output_lines[-1]
```

Comment:
I don't know if this is robust enough. What if a user has the privileges to execute multiple commands via sudo. The rule set can be longer than just having one entry, which this implementation assumes. In this case `[-2]` and `[-1]` does not work properly.

Reply:
true.